VisualC/pkg-support/cmake/sdl3_mixer-config.cmake

```diff
@@ -0,0 +1,78 @@
+# SDL3_mixer CMake configuration file:
+# This file is meant to be placed in a cmake subfolder of SDL3_mixer-devel-3.x.y-VC
+
+include(FeatureSummary)
+set_package_properties(SDL3_mixer PROPERTIES
+    URL "https://www.libsdl.org/projects/SDL_mixer/"
+    DESCRIPTION "SDL_mixer is a sample multi-channel audio mixer library"
+)
+
+cmake_minimum_required(VERSION 3.0)
+
+set(SDL3_mixer_FOUND                TRUE)
+
+set(SDL3MIXER_VENDORED              TRUE)
+
+set(SDL3MIXER_CMD                   FALSE)
+
+set(SDL3MIXER_FLAC_LIBFLAC          FALSE)
+set(SDL3MIXER_FLAC_DRFLAC           TRUE)
+
+set(SDL3MIXER_GME                   FALSE)
```

Comment:
Same here, assume the VC SDK does not have gme support.

Reply:
No, it doesn't.